NEWS.md

```diff
@@ -43,7 +43,7 @@
 
     * Now supports type `complex`, [#3690](https://github.com/Rdatatable/data.table/issues/3690).
 
-    * Gains `scipen` [#2020](https://github.com/Rdatatable/data.table/issues/2020), the number 1 most-requested feature [#3189](https://github.com/Rdatatable/data.table/issues/3189). The default is `getOption("scipen")` so that `fwrite` will now respect this R option in the same way as `base::write.csv` and `base::format`, as expected. The parameter and option name have been kept the same as base R's `scipen` for consistency and to aid online search. It stands for 'scientific penalty'; the number of characters to add to the width within which non-scientific number format is used if it will fit. A high penalty essentially turns off scientific format.
+    * Gains `scipen` [#2020](https://github.com/Rdatatable/data.table/issues/2020), the number 1 most-requested feature [#3189](https://github.com/Rdatatable/data.table/issues/3189). The default is `getOption("scipen")` so that `fwrite` will now respect this R option in the same way as `base::write.csv` and `base::format`, as expected. The parameter and option name have been kept the same as base R's `scipen` for consistency and to aid online search. It stands for 'scientific penalty'; the number of characters to add to the width within which non-scientific number format is used if it will fit. A high penalty essentially turns off scientific format. The efficiency-conscious should note that there will be a memory as well as a speed penalty to using large `scipen` values since we use this to allocate space for numeric fields; users with efficiency concerns but a need for high precision should set a generous but not arbitrarily large `scipen` according to their use case.
```

Comment:
Nice, but yet could be more precise by giving an example.
I can imagine a user want to have a guarantee that scientific notation won't be stored in a csv, so it will be more portable. They can now use `format(x, sci=F)`.
Does `round(x, 10)` and then `fwrite(scipen=10)` resolves that?